lab-06/quiz/template-specialisations.md

```diff
@@ -0,0 +1,10 @@
+## Question Text
+
+Why cannot we use template specialisations to aggregate all numeric types?
```

Comment:
I think this question is a bit odd. You could technically use template specializations, but then you would have to have an overload for each tree of types. That makes both the second and the third answers vaguely correct. Maybe you should spell out the fact that you want a single template declarations and not multiple overloads.